server/controllers/rpaFrameworkCommandsController.js

```diff
@@ -47,12 +47,13 @@ exports.getVarsForTask = async (req, res) => {
           Task: task,
         },
         {
-          InputVars: 1,
-          Output: 1
+          inputVars: 1,
+          outputValue: 1,
```

Comment:
These changes were necessary to adapt the call to our databse schema, so it can be tested with data similiar to the Db